### DISCUSSION TITLE
Reset After or Card Timout

### DISCUSSION THREAD

**Discussion by sumur1au** [kinghat/tabbed-card#79] [Feature Requests]
@kinghat thanks for the card. After struggling with the Swiper card, this card has been a saviour.  

One thing I would love is an reset after or timeout feature similar to the Swiper card.

**Comment by kinghat** [comment]
sounds useful. no promises but will keep it in mind.

**Comment by Mariusthvdb** [comment]
since you're mentioning Swipe card:

did I overlook a swipe option in the tabbed-card? I mean, ofc we have tabs, but still I find myself trying to swipe in Mobile.....

would be cool if possible

**Reply by kinghat** [reply]
ive thought about it but i think the swipe card handles that specific case well. while a smaller target, you are already able to scroll/swipe the tab bar. if i ever entertain the idea, ill post about it here.

**Reply by Mariusthvdb** [reply]
right, I wasn't aware of that, (though I did see it happening when using text labels) since I am using icons only. 

With 4 icons, there is nothing to swipe in the tab bar ;-)

hope you will revisit someday, thanks!

**Reply by kinghat** [reply]
> With 4 icons, there is nothing to swipe in the tab bar ;-)

good point. its not a no, but probably not in the near future.
